op-chain-ops/crossdomain/migrate.go

```diff
@@ -77,12 +79,15 @@ func MigrateWithdrawal(withdrawal *LegacyWithdrawal, l1CrossDomainMessenger *com
 		return nil, fmt.Errorf("cannot abi encode relayMessage: %w", err)
 	}
 
+	// Set the outer gas limit. This cannot be zero
+	gasLimit := uint64(len(data)*16 + 200_000)
+
 	w := NewWithdrawal(
 		withdrawal.Nonce,
 		&predeploys.L2CrossDomainMessengerAddr,
 		l1CrossDomainMessenger,
 		value,
-		new(big.Int),
+		new(big.Int).SetUint64(gasLimit),
```

Comment:
Is this the gas limit that is eventually passed to call which will be executed on L1?

Reply:
Yup exactly

Reply:
Got it. I also just saw the doc you wrote up